tentacle/src/service.rs

```diff
@@ -536,25 +420,33 @@ where
             // Send data to the specified protocol for the specified session.
             TargetSession::Single(id) => {
                 if let Some(control) = self.sessions.get_mut(&id) {
-                    control.push_message(proto_id, priority, data);
-                    control.try_send(cx);
+                    control.inner.incr_pending_data_size(data.len());
```

Comment:
Better `incr_pending_data_size` inside `control.send()` (by destruct `SessionEvent::ProtocolMessage`) in case we don't forget it before send `ProtocolMessage`.

Reply:
`incr_pending_data_size` only exists in this function, there is no need for the operation of this function to cause all other commands to also carry the destruct operation